Dockerfile

```diff
@@ -31,7 +31,7 @@ ENV PATH="$PATH:/home/$USERNAME/.local/bin" \
 
 # Install Python dependencies
 COPY requirements-dev.txt .
-RUN pip install -r requirements-dev.txt
+RUN pip install --user -r requirements-dev.txt
```

Comment:
Why install `--user` in a Dockerfile - I usually assume everything can go system-wide within the image..?

Reply:
At one point I was having some nasty ownership issues so I shifted to this, though I'm not sure if this was actually the fix in the end. Docker ownership stuff scares me, so this does make me feel a little safer even if it's not necessary 🤷 unless you think this could cause issues...